plugin/src/main/java/net/pl3x/map/configuration/Advanced.java

```diff
@@ -964,32 +970,31 @@ Setting a color to black (#000000) will make it invisible.""")
         put(Blocks.SUGAR_CANE, 0x95C165);
         put(Blocks.SUNFLOWER, 0xF6C436);
         put(Blocks.SWEET_BERRY_BUSH, 0x305E3A);
-        put(Blocks.TALL_GRASS, 0x000000);
         put(Blocks.TALL_SEAGRASS, 0x2E7604);
         put(Blocks.TARGET, 0xE2AA9E);
         put(Blocks.TERRACOTTA, 0x985E44);
         put(Blocks.TINTED_GLASS, 0x2B262E);
         put(Blocks.TNT, 0x873D36);
-        put(Blocks.TORCH, 0xC4A767);
+        put(Blocks.TORCH, 0x000000);
         put(Blocks.TRAPPED_CHEST, 0x866025);
-        put(Blocks.TRIPWIRE, 0x000000);
         put(Blocks.TRIPWIRE_HOOK, 0x000000);
-        put(Blocks.TUBE_CORAL, 0x3054C5);
+        put(Blocks.TRIPWIRE, 0x000000);
         put(Blocks.TUBE_CORAL_BLOCK, 0x3158CF);
         put(Blocks.TUBE_CORAL_FAN, 0x335BD1);
         put(Blocks.TUBE_CORAL_WALL_FAN, 0x335BD1);
+        put(Blocks.TUBE_CORAL, 0x3054C5);
         put(Blocks.TUFF, 0x6C6D67);
         put(Blocks.TURTLE_EGG, 0xE5E3C0);
-        put(Blocks.TWISTING_VINES, 0x14907D);
         put(Blocks.TWISTING_VINES_PLANT, 0x14897A);
+        put(Blocks.TWISTING_VINES, 0x14907D);
         put(Blocks.VERDANT_FROGLIGHT, 0xEAF6E9);
         put(Blocks.VINE, 0x747474);
         put(Blocks.VOID_AIR, 0x000000);
-        put(Blocks.WALL_TORCH, 0xC4A767);
+        put(Blocks.WALL_TORCH, 0x000000);
```

Comment:
I think torches should be visible, too. All 3 variants of them (normal, redstone, soul)

Reply:
Are you sure?

I imagine big areas completely plastered with torches to prevent mob spawning.
a torch is a pretty small part of a block